keps/sig-api-machinery/NNNN-crd-validation-defaulting-conversion-expressions/README.md

```diff
@@ -261,7 +257,7 @@ kind: CustomResourceDefinition
       properties:
         spec:
           x-kubernetes-validator: 
-		    - rule: "minReplicas <= maxReplicas"
+		    - rule: this.properties.minReplicas <= this.properties.maxReplicas
```

Comment:
Let's keep this as minimal as possible for now.  We might need a `this` keyword of equivalent, but I don't that's decided. And adding `properties` adds a lot of verbosity for unclear gains.